spec/hardware/main.fmf

```diff
@@ -19,6 +19,26 @@ description: |
     grouped under a common parent, for example ``cores`` or
     ``model`` under the :ref:`/spec/hardware/cpu` key.
 
+    Operators
+    ---------
+
+    The ``value`` part of the constraint follows the schema ``[operator] actual-value``.
```

Comment:
Could we mention it is advised to quote the constraint and the value? And is it?

Reply:
I'm not sure we can enforce it, it's behind a YAML and if the user omits quotes, tmt would never know, but we can definitely recommend using them. Added in 69169daa.

Reply:
@thrix yq is strange. other yaml parsers (e.g. ruamel):

```
YAML(typ='safe').load("memory: != 8 GB")
ConstructorError: could not determine a constructor for the tag '!='
```

Reply:
indeed ....